js/templates/modals/about/license.html

```diff
@@ -0,0 +1,37 @@
+<div class="contentBox padMd clrP clrBr clrSh3">
+  <div class="flexSpaceBetween">
+    <div class="flexRow flex0">
+      <div class="brandSmall" style="background-image: url('../imgs/baseRounded.svg');"></div>
+      <div>
+        <div><%= ob.polyT('about._brand') %></div>
+        <div class="flexRow">
+          <img src=1>
+          <img src=2>
+          <img src=3>
+        </div>
+      </div>
+    </div>
+    <h2 class="h3 clrT"><%= ob.polyT('about.licenseTab._sectionHeader') %></h2>
+    <div class="txtRight">
+      <div><%= ob.polyT('about.version') %> <%= ob.polyT('about.versionCode') %></div>
+      <a class="btn clrP clrBr js-copyAddress flexNoShrink">
```

Comment:
I think the `js-copyAddress` is a copy and paste error since semantically it doesn't make sense here.